streams/au.m3u

```diff
@@ -41,6 +41,8 @@ https://c.mjh.nz/101002210220
 https://bk7l2pn7dx53-hls-live.5centscdn.com/austamil/fe01ce2a7fbac8fafaed7c982a04e229.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Race Central TV (720p)
 https://nrpus.bozztv.com/36bay2/gusa-racecentral/index.m3u8
+#EXTINF:-1 tvg-id="TickerNews.au" status="online",Ticker News (1080p)
```

Comment:
The case of letters matters.

```suggestion
#EXTINF:-1 tvg-id="tickerNews.au" status="online",Ticker News (1080p)
```
https://iptv-org.github.io/channel?id=tickerNews.au